[NetworkUPSTools] Unexpected problem updating configuration

On OpenHAB startup the following java error is logged:

```
2018-07-09 21:50:02.222 [ERROR] [org.apache.felix.configadmin        ] - [org.osgi.service.event.EventHandler, org.osgi.service.cm.ManagedService, id=320, bundle=203/mvn:org.openhab.binding/org.openhab.binding.networkupstools/1.12.0]: Unexpected problem updating configuration org.openhab.networkupstools
java.lang.ArrayIndexOutOfBoundsException: 1
	at org.openhab.binding.networkupstools.internal.NetworkUpsToolsBinding.updated(NetworkUpsToolsBinding.java:167) [203:org.openhab.binding.networkupstools:1.12.0]
	at org.apache.felix.cm.impl.helper.ManagedServiceTracker.updated(ManagedServiceTracker.java:189) [8:org.apache.felix.configadmin:1.8.16]
	at org.apache.felix.cm.impl.helper.ManagedServiceTracker.updateService(ManagedServiceTracker.java:152) [8:org.apache.felix.configadmin:1.8.16]
	at org.apache.felix.cm.impl.helper.ManagedServiceTracker.provideConfiguration(ManagedServiceTracker.java:85) [8:org.apache.felix.configadmin:1.8.16]
	at org.apache.felix.cm.impl.ConfigurationManager$ManagedServiceUpdate.provide(ConfigurationManager.java:1479) [8:org.apache.felix.configadmin:1.8.16]
	at org.apache.felix.cm.impl.ConfigurationManager$ManagedServiceUpdate.run(ConfigurationManager.java:1435) [8:org.apache.felix.configadmin:1.8.16]
	at org.apache.felix.cm.impl.UpdateThread.run0(UpdateThread.java:141) [8:org.apache.felix.configadmin:1.8.16]
	at org.apache.felix.cm.impl.UpdateThread.run(UpdateThread.java:109) [8:org.apache.felix.configadmin:1.8.16]
	at java.lang.Thread.run(Thread.java:748) [?:?]
```

The items are then created by OpenHAB, but do not work (values not updated).
It makes no difference whether networkupstools.cfg is empty or fully configured.

My configuration:
- Debian stable AMD64
- openHAB 2.3.0-1 with matching addons from https://dl.bintray.com/openhab/apt-repo2
- openjdk 1.8.0_171

--- a/bundles/binding/org.openhab.binding.networkupstools/src/main/java/org/openhab/binding/networkupstools/internal/NetworkUpsToolsBinding.java
+++ b/bundles/binding/org.openhab.binding.networkupstools/src/main/java/org/openhab/binding/networkupstools/internal/NetworkUpsToolsBinding.java
@@ -163,6 +163,10 @@ public class NetworkUpsToolsBinding extends AbstractActiveBinding<NetworkUpsTool
                 }
 
                 String[] parts = key.split("\\.");
+                if (parts.length < 2) {
+                    throw new ConfigurationException(key, "Invalid configuration setting."
+                    	+" Configuration key must contain a period ('.'), but does not.");
+                }
                 String name = parts[0];
                 String prop = parts[1];
                 String value = (String) config.get(key);

